### PR TITLE
docs: add "Component Events from NGXS" recipe

### PR DESCRIPTION
**Pull request by arturovt**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
   * [Authentication](recipes/authentication.md)
   * [Caching](recipes/cache.md)
   * [Debouncing Actions](recipes/debouncing-actions.md)
+  * [Actions as DOM Events](recipes/actions-as-dom-events.md)
```
Comment (markwhitfeld) [inline docs/SUMMARY.md]: These seem to be alphabetical

Comment (arturovt) [reply docs/SUMMARY.md]: `Actions As DOM Events`? or prolly I'm missing something... :smile:

**Comment by arturovt** [comment]
@markwhitfeld I'm flexible with any name for this recipe.

File: docs/SUMMARY.md (hunk 1 above)
Comment (markwhitfeld) [reply docs/SUMMARY.md]: The menu items seem to be alphabetical and this item is then out of order.

Comment (arturovt) [reply docs/SUMMARY.md]: Gotcha

**Comment by arturovt** [comment]
@markwhitfeld should we merge?

**Comment by arturovt** [comment]
@markwhitfeld renamed to "Emitting DOM Events via NGXS"

**Comment by markwhitfeld** [comment]
How about "Component Events from NGXS" or "Component Event Source" or "Output Event Source"?

**Comment by splincode** [comment]
`Output Event Source` - pretty cool

**Comment by arturovt** [comment]
Gentlemen, I guess the `Component Events from NGXS` is pretty good name because it describes more clearly what the user might see in the article. Are we OK with it or maybe there are some ideas? :D

**Comment by splincode** [comment]
@arturovt cool, also fix merge conflict

**Pull request merged**